test/Dialect/Torch/canonicalize.mlir

```diff
@@ -1253,6 +1253,15 @@ func.func @torch.aten.Int.Tensor(%arg0: !torch.int) -> !torch.int {
   return %scalar : !torch.int
 }
 
+// CHECK-LABEL:   func.func @torch.aten.Int.float() -> !torch.int {
+// CHECK:             %[[NUM:.*]] = torch.constant.int 1
+// CHECK:             return %[[NUM]] : !torch.int
+func.func @torch.aten.Int.float() -> !torch.int {
```

Comment:
Can you also add an e2e test?

Reply:
Hi Ramiro, it seems hard to add a MHLO conversion for a scalar integer to enable a e2e test. Also, the previous folder PRs all only do canonicalize test: https://github.com/llvm/torch-mlir/pull/1691, https://github.com/llvm/torch-mlir/pull/1257

Reply:
I add the e2e test. The torch.constant.int‘s value attribute is Singed Integer, while the  arith.constant only accept Signless Integer. So I add the type conversion when lowering torch.constant.int to arith.constant.

Reply:
The CI failed in strange way. Could you rerun it? Thanks.